docs/index.markdown

```diff
@@ -28,27 +28,24 @@ The New Relic Terraform Provider enables Observability as Code, reducing develop
 * [Documentation](https://www.terraform.io/docs/providers/newrelic/index.html)
 * [Repo](https://github.com/terraform-providers/terraform-provider-newrelic/)
 * [Project boards](https://github.com/terraform-providers/terraform-provider-newrelic/projects)
-* Roadmap
 
 
-#### New Relic API Client
+#### New Relic Client
 
 The New Relic Client provides the building blocks for many tools in the toolkit, enabling quick access to the suite of New Relic APIs.  As a library, it can also be leveraged within your own custom applications.
 
 * Documentation
 * Repo
 * Project Boards
-* Roadmap
 
 
 #### New Relic CLI
 
-The New Relic CLI allows integrating New Relic into your existing workflows. Be it fetching data from your laptop while troubleshooting an issue, or adding New Relic into your CI/CD pipeline, the `nr-cli` will be there
+The New Relic CLI allows integrating New Relic into your existing workflows. Be it fetching data from your laptop while troubleshooting an issue, or adding New Relic into your CI/CD pipeline.
```

Comment:
suggest tweaking a bit - "The New Relic CLI enables integration of New Relic into your existing workflows"